kofta/public/locales/cs/translation.json

```diff
@@ -138,23 +139,23 @@
       "addSupport": "a pokusíme se podporu přidat."
     },
     "inviteButton": {
-      "invited": "pozván",
-      "inviteToRoom": "pozvat do místnosti"
+      "invited": "Pozván",
+      "inviteToRoom": "Pozvat do místnosti"
     },
     "micPermissionBanner": {
       "permissionDenied": "Nemáš povolený mikrofon! Zkus znovu načíst stránku.",
-      "dismiss": "zavřít",
-      "tryAgain": "zkusit znovu"
+      "dismiss": "Zavřít",
+      "tryAgain": "Zkusit znovu"
     },
     "keyboardShortcuts": {
-      "setKeybind": "nastavit klávesovou zkratku",
-      "listening": "posluchám",
-      "toggleMuteKeybind": "ztlumit mikrofon",
-      "togglePushToTalkKeybind": "Pust-To-Talk",
-      "toggleOverlayKeybind": "toggle overlay keybind"
+      "setKeybind": "Nastavit klávesovou zkratku",
+      "listening": "Poslouchám",
+      "toggleMuteKeybind": "Ztlumit mikrofon",
+      "togglePushToTalkKeybind": "Push-To-Talk (mluvit jen po zmáčknutí tlačítka)",
+      "toggleOverlayKeybind": "Nastavit klávesovou zkratku pro overlay"
     },
     "userVolumeSlider": {
-      "noAudioMessage": "z nějakého důvodu se nepřehrává zvuk"
+      "noAudioMessage": "Z nějakého důvodu není vysílán zvuk..."
```

Comment:
```suggestion
      "noAudioMessage": "Z nějakého důvodu není vysílán zvuk"
```
what is this for?

Reply:
no audio consumer for some reason

This happens when no audio consumer is spawned by MediaSoup on the backend, usually because the particular person isn't streaming any.

This usually means the person is a bot.